go.mod

```diff
@@ -8,17 +8,18 @@ require (
 	github.com/enescakir/emoji v1.0.0
 	github.com/fatih/color v1.14.1
 	github.com/inhies/go-bytesize v0.0.0-20220417184213-4913239db9cf
+	github.com/jayco/go-emoji-flag v0.0.0-20190810054606-01604da018da
 	github.com/mattn/go-runewidth v0.0.14
-	github.com/onsi/ginkgo/v2 v2.8.4
+	github.com/onsi/ginkgo/v2 v2.9.0
```

Comment:
Can you please also run `licenses/generate-notice-file.sh` ?

Reply:
Done.